journal/src/test/java/io/zeebe/journal/file/JournalTest.java

```diff
@@ -374,6 +392,31 @@ public void shouldReadReopenedJournal() throws Exception {
     assertThat(reader.next()).isEqualTo(appendedRecord);
   }
 
+  @Test
+  public void shouldWriteToReopenedJournalAtNextIndex() throws Exception {
+    // given
+    final var firstRecord = journal.append(data);
+    journal.close();
+    assertThat(journal.isOpen()).isFalse();
```

Comment:
Same as above - if we have a test that `journal.close()` closes properly, it will fail should it not work, no need to test here as well unless we think the error we get below is very hard to diagnose.